manifests/init.pp

```diff
@@ -136,7 +136,7 @@
   $redis_slave_output_buffer_soft_limit = $redis::params::redis_slave_output_buffer_soft_limit,
   $redis_slave_output_buffer_soft_limit_max_interval = $redis::params::redis_slave_output_buffer_soft_limit_max_interval,
   $redis_snapshotting = $redis::params::redis_snapshotting,
-  $manage_config_file = $redis::params::manage_config_file
```

Comment:
OH, manage_config_files is our own voodoo?  what was it there for?    I'm guessing we have a need to install an initial config and then let redis/sentinel update it with dynamic changes.. ? 

hmm. I still think something is off here.  but in order to not block, I wilthdraw my objections for the time being.